jumpscale/packages/marketplace/chats/exposed.py

```diff
@@ -68,27 +68,27 @@ def domain(self):
             retry = False
             while True:
                 domain = self.string_ask(
-                    f"Please specify the sub domain name you wish to bind to. will be (subdomain).{self.domain}",
+                    f"Please specify the sub domain name you wish to bind to. This will be a (subdomain).{self.domain}",
```

Comment:
the example is shown here , so probably should be `This will be (subdomain).{self.domain}` without "a"

Reply:
a beginning of the sentence after a "." should ALWAYS have big first letter. 

example:

I do this sentence. Now the first letter in 'now' on the second sentence is always big.